lib/template.html

```diff
@@ -89,7 +92,7 @@
 {% endif %}
 
 {% if page.image %}
-  {% assign seo_page_image = page.image.path | default: page.image.facebook | default: page.image %}
+  {% assign seo_page_image = page.image.path | default: page.image.url | default: page.image.facebook | default: page.image %}
```

Comment:
Allow for page.image.url here, else there will be a mess of if interpreting the yaml object when it encounters this unknown yaml field.